### PR TITLE
Add polling and device isolation

### PR DESCRIPTION
**Pull request by dlisiewicz**

### CODE REVIEW

File: wsapp_connector.py [hunk 1]
```diff
@@ -262,6 +276,150 @@ def _handle_get_events(self, param):
...
+                severity = i['severity']
+                artifacts.append({"name": 'Event '+str(k),  "label": severity,  "severity": severity,
+                                  'artifact_type': 'event ', "data": i, "cef": i, "container_id": 224, "run_automation": True, 'tags': 'new'})
+                k+=1
+        if len(artifacts) >=1:
+            success, message, id_list = self.save_artifacts(artifacts)
+
+        # Add the response into the data section
+        
+        print("Liczba Eventów: ", len(action_result.get_data()))
```
Comment (brusju) [inline wsapp_connector.py]: Please write this message in English so we are consistent.

File: wsapp.png
Comment (brusju) [inline wsapp.png:1]: I think we should stick to APP placeholder.

File: wsapp_dark.png
Comment (brusju) [inline wsapp_dark.png:1]: Same thing here, let's stick with APP placeholder.

**Review by brusju — CHANGES_REQUESTED** [review]
Good work, just some minor requests from me.

File: wsapp_connector.py [hunk 2]
```diff
@@ -19,7 +20,7 @@
 from bs4 import BeautifulSoup
 from oauthlib.oauth2 import BackendApplicationClient
 from requests_oauthlib import OAuth2Session
```
Comment (wozprzemek) [inline wsapp_connector.py]: nit: I don't see these used anywhere, let's not import unused things

File: wsapp_connector.py [hunk 3]
```diff
@@ -262,6 +276,150 @@ def _handle_get_events(self, param):
...
+        client_id = self._client_id
+        client_secret = self._client_secret
+        base_url = self._base_url
+        token = self._get_token(client_id, client_secret, base_url)
+        headers = {'Authorization': f'Bearer ' + token}
+        ret_val, response = self._make_rest_call(
+            'whoami/v1/whoami', action_result, params=None, headers=headers
+        )
+        organizationId = response["organizationId"]
+        info = self.get_container_info(container_id=224)
```
Comment (wozprzemek) [inline wsapp_connector.py:301]: Will this hardcoded id always remain the same?

Comment (Jnkess) [reply wsapp_connector.py:301]: Currently it needs to be hardcoded.

**Review by wozprzemek — APPROVED** [review]

**Review by brusju — APPROVED** [review]

**Pull request merged**